sw/device/lib/dif/dif_rstmgr.c

```diff
@@ -126,7 +112,11 @@ dif_rstmgr_result_t dif_rstmgr_reset_lock(const dif_rstmgr_t *handle,
   }
 
   mmio_region_t base_addr = handle->params.base_addr;
-  rstmgr_software_reset_lock(base_addr, peripheral);
+  uint8_t index = rstmgr_regen_rstctrl_bit_index(peripheral);
+  uint32_t bitfield = bitfield_bit32_write(0, index, false);
```

Comment:
`bitfield == 0` unconditionally here I think? Did you mean to do `~bit32_write(..., true);`?

Reply:
Good point, thanks!

Done.